softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/AbstractCommandRouterHandler.java

```diff
@@ -0,0 +1,21 @@
+package com.softgroup.common.router.api;
+
+import com.softgroup.common.protocol.Request;
+
+/**
+ * Created by User on 21.03.2017.
+ */
+public abstract class AbstractCommandRouterHandler<T extends Handler> extends AbstractRouterHandler<T> {
+
+
+    public abstract String getName();
```

Comment:
@AlexBort this method is declared in interface. Why you need to re-declare it here ?